go.mod

```diff
@@ -5,24 +5,24 @@ go 1.18
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
-	github.com/cosmos/cosmos-sdk v0.46.2
-	github.com/cosmos/ibc-go/v5 v5.0.0
+	github.com/cosmos/cosmos-sdk v0.46.4
+	github.com/cosmos/ibc-go/v5 v5.0.1
 	github.com/ethereum/go-ethereum v1.10.19
-	github.com/evmos/ethermint v0.6.1-0.20221003153722-491c3da7ebd7
+	github.com/evmos/ethermint v0.6.1-0.20221101220534-a8ea4eceb6d9
```

Comment:
https://github.com/evmos/ethermint/compare/491c3da7ebd7...a8ea4eceb6d9

Reply:
it contains a breaking change, it add support to multisig of ethsecp256k1 keys, maybe ok to smuggle into testnet?

Reply:
switched to a fork, which reverted the ethsecp256k1 multisig support.